Update notification texts in Bring (#37395)
home-assistant/home-assistant.github.io: source/_integrations/bring.markdown

If you want to receive these notifications, you must use a dedicated account, as outlined in the [known limitations](#known-limitations).
{% endnote %}

| Data attribute         | Optional | Description                                                                                                                             |
| ---------------------- | -------- | --------------------------------------------------------------------------------------------------------------------------------------- |

> | `target`               |       no | Target Bring! list(s) the members to be notified.                                                                                 |

Reviewer: ```suggestion
| `target`               |       no | Target Bring! list(s) whose members should be notified.                                                                                 |
```

> | `target`               |       no | Target Bring! list(s) whose members should be notified.                                                                                 |

Reviewer: ```suggestion
| `target`               |       no | Target Bring! list(s) the members to be notified.                                                                                 |
```
